content/spin/v2/quickstart.md

```diff
@@ -282,21 +279,23 @@ The additional `spin.toml` file is the manifest file, which tells Spin what even
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
+spin_manifest_version = "2"
 authors = ["You <you@yourself.net>"]
```

Comment:
We'll need to add `[application]` just above where we add authors, description, name and version.

eg https://github.com/fermyon/spin/blob/main/examples/http-rust/spin.toml#L3-L7